### ISSUE TITLE
When the card is discarded, it will enter the Zone.GRAVEYARD

### ISSUE THREAD

**Issue by shinoi2** [jleclanche/fireplace#462]

**Comment by tonyyyye** [comment]
Yes I think you are right. The GRAVEYARD should store and only store all the DEAD minion, or the Resurrect BRM_017 will have a wrong choice range.

**Comment by jleclanche** [comment]
The problem is there is no such thing as the DISCARD zone, that was something I added later on. Hearthstone tracks a graveyard zone separately... we don't mimick that at this time. I dunno what to do about it.
